src/test/kotlin/com/github/burrunan/s3cache/RemoteCacheTest.kt

```diff
@@ -58,7 +58,7 @@ class RemoteCacheTest: BaseGradleTest() {
         private fun gradleVersionAndSettings(): Iterable<Arguments> {
             if (!isCI) {
                 // Use the current Gradle version to make the test faster
-                return listOf(arguments("4.1"))
+                return listOf(arguments("6.6"))
```

Comment:
This is now the current Gradle version as of writing :)

Reply:
The comment above `4.1` is not quite right.
The idea here is to use the minimal supported Gradle version for local testing, and CI adds much more versions (see below)
That enables fast local tests, and it is very easy to capture unexpected dependencies on Gradle 6 API